.github/workflows/ci.yml

```diff
@@ -49,6 +49,39 @@ jobs:
       - name: check namespacing
         run: |
           ./scripts/ci/check-namespace
+  quickcheck-c90:
+    strategy:
+      fail-fast: false
+      matrix:
+        external:
+         - ${{ github.repository_owner != 'pq-code-package' }}
+        target:
+         - runner: pqcp-arm64
+           name: 'aarch64'
+         - runner: ubuntu-latest
+           name: 'x86_64'
+        exclude:
+          - {external: true,
+             target: {
+               runner: pqcp-arm64,
+               name: 'aarch64'
+             }}
+    name: Quickcheck C90 (${{ matrix.target.name }})
+    runs-on: ${{ matrix.target.runner }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: make quickcheck
+        run: |
+          OPT=0 CPPFLAGS=-std=c90 make quickcheck >/dev/null
+          make clean            >/dev/null
+          OPT=1 CPPFLAGS=-std=c90 make quickcheck >/dev/null
+      - uses: ./.github/actions/setup-ubuntu
+      - name: tests func
+        run: |
+          CPPFLAGS="-std=c90" ./scripts/tests func
```

Comment:
If we use `tests`, we should use the `--cflags` argument

Reply:
Ok, @mkannwischer explained that this does not work at the moment because `std=c90` would end up before `std=c99` on the cmd line, and `c99` would take precedence